utils/luci-app-ffwizard-berlin/root/usr/share/rpcd/acl.d/luci-app-ffwizard-berlin.json

```diff
@@ -0,0 +1,80 @@
+{
+        "luci-app-ffwizard-berlin": {
+                "description": "Provides access to config files and scripts",
+                "read": {
+                        "cgi-io": [
+				"exec"
+			],
+                        "uci": [
+                                "profile_*",
+                                "system",
+                                "ffwizard",
+                                "dhcp",
+                                "olsrd",
+                                "olsrd6",
+                                "firewall",
+                                "freifunk",
+                                "wireless",
+                                "network",
+                                "qos",
+                                "luci_statistics",
+                                "openvpn"
+
+                        ],
+                        "files" : {
```

Comment:
Do we really need all this permissions? 
Why having "/etc/config/ffwizard" listed explicitly

Reply:
Because we access this file directly and not via uci. I don't know why... Just looked into the code which files are accessed.